Packs/Base/Scripts/CommonServerPowerShell/CommonServerPowerShell.ps1

```diff
@@ -546,13 +546,13 @@ End {
                 {
                     if ($null -ne $raw_value)
                     {
-                        if ($raw_value -Is [System.Array] -Or $raw_value -Is [Collections.IDictionary] -Or $raw_value -Is [PSCustomObject])
+                        if ($raw_value -is [int] -or $raw_value -is [string] -or $raw_value -is [int] -or $raw_value -is [Double])
```

Comment:
why are we checking here twice for `int`? also there are many types here that are not covered, such as `float`
did you consider using regex?

Reply:
Done